src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/QueryFunctionsTest.java

```diff
@@ -212,6 +212,54 @@ public void numberLiteralInSelectField() {
         );
     }
 
+    @Test
+    public void ifFunctionWithConditionStatement() {
+        String query = "SELECT IF(age > 35, 'elastic', 'search') AS Ages FROM accounts";
+        ScriptField scriptField = CheckScriptContents.getScriptFieldFromQuery(query);
+        assertTrue(
+                CheckScriptContents.scriptContainsString(
+                        scriptField,
+                        "boolean cond = doc['age'].value > 35;"
+                )
+        );
+    }
+
+    @Test
+    public void ifFunctionWithEquationConditionStatement() {
+        String query = "SELECT IF(age = 35, 'elastic', 'search') AS Ages FROM accounts";
```

Comment:
Is there any way to assert that the alias `Ages`  is being properly generated?

Reply:
Yes like by comparing it with a proper expected output file. But I have tested the cases where the condition expression does not contain any column from a table, and the output is proper. Unit test indicates the script for the function is generated properly as well. I guess it's quite enough so the result could be exempted? Btw I also sanity tested seeveral cases. What do you think?